editoast/openapi_legacy.yaml

```diff
@@ -394,6 +394,28 @@ paths:
                     items:
                       $ref: "#/components/schemas/InfraError"
 
+  /infra/{id}/auto_fix/:
```

Comment:
Maybe instead of writing it to the legacy openapi, you could directly write utoipa anotations? (might be a mess depending on where Operation is defined

Reply:
The Create operation takes a `RailjsonObject` which references like 90% of the structs of schemas 😁 I mean, if you're more than welcome to do it if you feel like it, but that's a fairly large amount of work that might sidetrack this PR.

Reply:
:zipper_mouth_face:

Reply:
Thanks to @Tristramg we were able to get around `RailjsonObject` migration for now, and a separate PR is open for that: https://github.com/osrd-project/osrd/pull/5820
I resolve this thread to move discussions there, but reopen it if needed.